test/data/csp-nonce-external.js

```diff
@@ -0,0 +1,5 @@
+/* global startIframeTest */
+
+jQuery( function() {
+	$( "body" ).append( "<script nonce='jquery+hardcoded+nonce' src='csp-nonce.js'><\x2Fscript>" );
```

Comment:
The `<\x2Fscript>` hack is not necessary - it's meant to avoid HTML parsing quirks and this is an external script.